### ISSUE TITLE
Add. 17,215, foll. 30–33 | https://bl.vuexistapps.us/manuscript/903 (Part10)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#797]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- The sentence in Hand 2 has been repeated two other times in Notes and Additions. 
- Hand 2: Lack of space in "theverso"
- Hand 2: Lack of space in "leafhas"
- The sentence in Hand 2 informs that some parts of the writing were eliminated to make room for some verses from the gospel of Mark. Does it mean that we are dealing with a second hand? Did the main scribe efface his/her own writing to write something else? If yes, there is no Hand 2. Or did another person efface the first scribe's writing to add the final words of Mark's gospel? If yes, shouldn't the verso of the last leaf be considered a palimpsest? 
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [X] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
@wlpotter 
Should we keep hand 2 given the question above?
Should we change hand2 to "see additions" and delete the sentence from note under item 1?

**Comment by wlpotter** [comment]
Yes, let's keep this as a second hand as it is unlikely a scribe would erase their own writing to replace it with different contents (sometimes you see sections erased and redone if there was a mistake, but that doesn't seem to be the case here).

In addition, can you add a second msItem as a sibling to the first?
- It should have the title of Gospel of S. Mark, ch. xvi. 19, 20
- Can have the locus of f. 33b
- The note in ms item 1 should just read: The greater part of the writing on the verso of the last leaf has been, effaced
- Both the first and second items should be given `@defective="true"`
- The addition can stay as-is

**Comment by mikey-tang16** [comment]
msitem 2 added accordingly
